Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -217,9 +217,9 @@ open class XCTWaiter {
         self.manager = manager
 
         // Begin the core wait loop.
-        let timeoutTimestamp = CFAbsoluteTimeGetCurrent() + timeout
+        let timeoutTimestamp = Date.timeIntervalSinceReferenceDate + timeout
```

Comment:
Alternatively (and perhaps more idiomatically), this could do something like `let timeoutDate = Date(timeIntervalSinceNow: timeout)`, and then below inspect the result of `timeoutDate. timeIntervalSinceNow`.

Reply:
The same idea could apply to the cases below in the tests as well - working in terms of `Date` rather than `TimeInterval`. It's not critical, though, and I would be ok with this staying as-is.

Reply:
I may try to do that as a follow up.